src/components/FindPark.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <section>
     <h2>Let's Go Play!</h2>
-    <router-link to='/results'>
+    <router-link
+      :class="{ disabled: !this.$store.state.geolocation }"
+      to="/results"
+    >
       <button
         :class="{ disabled: !this.$store.state.geolocation }"
```

Comment:
Do we want this button to be disabled if `geolocation` is `null`? I don't believe this part of the functionality depends on `geolocation` since the user is providing input rather than clicking the button to get nearby parks.